crates/store/re_datafusion/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+//! The Rerun public data APIs. Access `DataFusion` `TableProviders`.
+
+use std::sync::Arc;
+
+use datafusion::catalog::TableProvider;
+use partition_list::PartitionListProvider;
+use re_log_types::external::re_tuid::Tuid;
+use re_protos::{
+    catalog::v1alpha1::{ext::EntryDetails, DatasetEntry, EntryFilter, ReadDatasetEntryRequest},
+    frontend::v1alpha1::frontend_service_client::FrontendServiceClient,
+};
+use table_entry_provider::TableEntryProvider;
+use tonic::transport::Channel;
+
+pub mod grpc_streaming_provider;
+pub mod partition_list;
+pub mod table_entry_provider;
+
+pub struct DataFusionConnector {
```

Comment:
nit: i'm growing increasingly uncomfortable with `lib.rs`/`mod.rs` containing anything else that published symbols.